lib/CL/devices/common.c

```diff
@@ -712,6 +712,11 @@ pocl_exec_command (_cl_command_node *node)
       POCL_UPDATE_EVENT_COMPLETE_MSG (event, "Event SVM Migrate_Mem       ");
       break;
 
+    case CL_COMMAND_COMMAND_BUFFER_KHR:
+      pocl_update_event_running (event);
```

Comment:
This doesn't seem right WRT timestamps etc. Needs comments describing what goes on.